src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -115,13 +130,54 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
         }
     }
 
-    deinitialize(): Promise<void> {
-        this._paymentMethod = undefined;
-        this._paypalCheckout = undefined;
+    private _renderVenmoButton( venmoInstance: VenmoInstance) {
+        const { venmoParentContainer } = this._renderButtonsData as RenderButtonsData;
+        const venmoButton = document.createElement('div');
```

Comment:
Just out of curiosity why not create `venmoButton` as a `button` element?

Reply:
We had styles for venmo block used div tag